src/services/anchor-service.ts

```diff
@@ -306,10 +306,12 @@ export default class AnchorService {
     }
 
     try {
-      const anchorCid = await this.ceramicService.publishAnchorCommit(
-        candidate.streamId,
-        ipfsAnchorCommit
-      )
+      // TODO(#548): Publish anchor commits via Ceramic
+      // const anchorCid = await this.ceramicService.publishAnchorCommit(
+      //   candidate.streamId,
+      //   ipfsAnchorCommit
+      // )
+      const anchorCid = await this.ipfsService.storeRecord(ipfsAnchorCommit)
```

Comment:
Why do we need to store the anchor commit? This commit would be created by js-ceramic nodes when they get the anchored successful response?

Reply:
The CAS just passes the anchor commit CID to the ceramic node that requested the anchor. The ceramic node then loads the anchor commit from ipfs, applies it, and publishes it to pubsub.

If the commit doesn't exist on ipfs then loading the commit on ceramic will fail

Reply:
Oh I see.